packages/tabs/src/_index.scss

```diff
@@ -0,0 +1,3 @@
+@forward './Default/styles.module.scss' as defaultTabs-*;
+@forward './Tab/styles.module.scss' as tabTabs-*;
```

Comment:
These `Tab` and `Test` forwards here are breaking the build. We're not using it anymore so you can take it out.